support-frontend/assets/pages/new-contributions-landing/components/ContributionFormFields.jsx

```diff
@@ -164,8 +164,21 @@ function FormFields(props: PropTypes) {
         formHasBeenSubmitted={checkoutFormHasBeenSubmitted}
         showRequiredLabel={showRequiredLabel}
       />
-      <details className={showRequiredLabel ? 'form-fields__required-explainer' : 'hidden'}>
-        <summary >
+      <details
+        className={showRequiredLabel ? 'form-fields__required-explainer' : 'hidden'}
+      >
+        <summary
+          onClick={(event) => {
+            if (event.detail !== 0) {
```

Comment:
Could you add a comment here explaining why we are checking this? Maybe with a link to the stack overflow question